src/gamecontroller.cs

```diff
@@ -447,6 +505,34 @@ partial interface GCMotion {
 		[TV (13,0), Mac (10,15), iOS (13,0)]
 		[Export ("setStateFromMotion:")]
 		void SetState (GCMotion motion);
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("hasAttitude")]
+		bool HasAttitude { get; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("hasRotationRate")]
+		bool HasRotationRate { get; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("sensorsRequireManualActivation")]
+		bool SensorsRequireManualActivation { get; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("sensorsActive")]
+		bool SensorsActive { get; set; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("hasGravityAndUserAcceleration")]
+		bool HasGravityAndUserAcceleration { get; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Export ("acceleration")]
+		GCAcceleration Acceleration { get; }
+
+		[TV (14,0), Mac (11,0), iOS (14,0)]
+		[Export ("setAcceleration:")]
+		void SetAcceleration (GCAcceleration acceleration);
```

Comment:
Method and a property? That is very weird? Acceleration and SetAcceleration do look like a perfect candidate for a property.